codecov.yml

```diff
@@ -1,7 +1,14 @@
 codecov:
   branch: master
 
-comment: off
+comment:
```

Comment:
Would we want to keep this `comments` field as is (looks like it's commenting in the PR which we also likely don't want) but add
```
github_checks:
  annotations: false
```

Reply:
But that would mean we only want to see codecov in the codecov dashboard and not anywhere in the PR's, right? I would say small report as above is not disturbing, but I am not aware of what exactly was agreed upon besides your post in gitter.

Reply:
I might be wrong, but I think from what @jorisvandenbossche was saying having comments off was on purpose, so here I'd assume we only want to turn off the annotations for untouched code as well

Reply:
Indeed, this was disabled on purpose (see https://github.com/pandas-dev/pandas/pull/27066) because those comments were most of the time noise, just as the annotations.